test/fhirtypes/ElementDefinition.assignBoolean.test.ts

```diff
@@ -119,32 +129,34 @@ describe('ElementDefinition', () => {
       );
       const clone = cloneDeep(userSelected);
       expect(() => {
-        userSelected.fixValue(true, true);
-      }).toThrow('Cannot fix true to this element; a different boolean is already fixed: false.');
+        userSelected.assignValue(true, true);
+      }).toThrow(
+        'Cannot assign true to this element; a different boolean is already assigned: false.'
+      );
       expect(clone).toEqual(userSelected);
     });
 
-    it('should throw FixedToPatternError when trying to change fixed[x] to pattern[x]', () => {
+    it('should throw AssignedToPatternError when trying to change fixed[x] to pattern[x]', () => {
```

Comment:
This is also super minor, but here and in a few other places, the test name references "AssignedToPatternError", which I think should still be "FixedToPatternError". Again, if this is too nit picky, that's fine. It is just in a test description so it doesn't effect anything.

Reply:
This one I actually think is fine, since the `FixedToPatternError` is thrown when we try to convert a `fixed[x]` value to a `pattern[x]` value, which cannot be done, so the error is for going from "Fixed to Pattern".